pull-request.sh

```diff
@@ -63,7 +64,7 @@ create_pull_request() {
     # Option 2: Open a new pull request
     else
         # Post the pull request
-        DATA="{\"title\":\"${TITLE}\", \"base\":\"${TARGET}\", \"head\":\"${SOURCE}\"}"
+        DATA="{\"title\":\"${TITLE}\", \"base\":\"${TARGET}\", \"head\":\"${SOURCE}\", \"draft\":\"${DRAFT}\"}"
         echo "curl --user ${GITHUB_ACTOR} -X POST --data ${DATA} ${PULLS_URL}"
```

Comment:
This looks good! In that draft must be in true | false (and correct me if this isn't the case) we don't protect for the case that the user incorrectly sets MAKE_DRAFT_PR to something wrong like "yes" or "no." A quick suggestion - instead of checking for a specific variable true/false, just have the total presence or absence indicate true or false, and set both these values directly in the script. So basically:

```bash
    if [ -z "${MAKE_DRAFT_PR}" ]; then
        echo "No explicit preference for draft PR; defaulting to normal PR."
        MAKE_DRAFT_PR="false"
    else
        MAKE_DRAFT_PR="true"
    fi
```
And then add a note to the README that the user can define the variable to be anything to turn on,  and leave unset to keep off. something like:

```bash
 **MAKE_DRAFT_PR**: set to anything (e.g., true/yes) to indicate that the pull request be a draft PR? (optional; unset defaults to `false`)
```
What we don't want to happen is an incorrectly set value being passed to the GitHub API and getting some invalid response. What do you think?

Reply:
Yeah, I like this - adding now!